k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -39,7 +39,7 @@ pub enum DeleteResources {
 /// Delete Upgrade resource.
 #[derive(clap::Args, Debug)]
 pub struct DeleteUpgradeArgs {
-    /// If set then upgrade will skip the io-engine pods restart
+   /// Upgrade to the specified branch.
```

Comment:
Why do we have/need upgrade_to_branch as part of DeleteUpgradeArgs struct?

Reply:
`upgrade_to_branch` is used to get the name of the upgrade resources. It is appended as suffix to the upgrade resource names. This is only for testing by developers as they can test whether the upgrade to any feature branch can happen or not. This flag is hidden to normal users.
Upgrade jobs are like : 
`mayastor-upgrade-job-develop`
`mayastor-upgrade-job-thin-provisioning`


Since the branch name is appended as suffix , we need to the branch name while deleting as well. This is needed in getting the exact name of the upgrade resource.